.travis.yml

```diff
@@ -109,12 +109,38 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - |-
+        sudo apt -yq --no-install-suggests --no-install-recommends install
+        g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib
+        libc6-arm64-cross
+      - build/linux/sysroot_scripts/install-sysroot.py --arch=arm64
+      - export DENO_BUILD_ARGS="target_cpu=\"arm64\" v8_target_cpu=\"arm64\""
+      - export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - export CC_aarch64_unknown_linux_gnu="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --locked
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --tests --locked
+      - |-
+        mv $(find $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release
+        -maxdepth 1 -type f -executable -regextype sed -regex
+        '.*/[a-z_]*-[0-9a-f]*') 
```

Comment:
What is this? I think this should be removed.

Reply:
This might need a comment. 
`cargo build --tests` builds `cargo test` as a executable
Unfortunately the name of the binary can't be specified. I think it's based on a hash or something.
The solution is to find said executable and rename it to test_rs.

Reply:
Leave out `cargo build --tests` for now. We don't run that elsewhere and it's going to require us to re-think our test setup.

Reply:
It shouldn't require any changes. The binary it outputs is almost identical to the `test_rs` from gn builds(different process same result). I've already successfully used it with `tools/test.py`.